### ISSUE TITLE
In case child state uses the same template as parent does, jQuery does not see it

### ISSUE THREAD

**Issue by dmytro-shchurov** [angular-ui/ui-router#1650]
After the latest update of ui-router `0.2.13` I faced to a detached DOM problem. I had a routing configuration when a parent and a child states reused the same template, which managed elements appearance depending on state param (because of a lot of layout elements I'd liked to reuse).
Everything worked fine in `0.2.11`, but not now. Entire child state view seems 'detached'... jQuery continues working with parent's view elements. In a browser console I run jQuery methods, but result is available only after I navigate back to a parent view.

**Comment by eddiemonge** [comment]
https://gist.github.com/eddiemonge/f6a58169c2846731a1eb
